src/components/Spinner/Spinner.module.scss

```diff
@@ -0,0 +1,106 @@
+.preloaderWrapper {
```

Comment:
Can we use something like bem here, but we can modify this.
For example the main CSS class should names like component - spinner and nested classes like .spinnerSomething